### PR TITLE
Maintenance Page: State, Mutations and Actions added

### PR DESCRIPTION
**Pull request by JuanJosePG**

### CODE REVIEW

**Comment by gonzalo-plaza** [comment]
@JuanJosePG For my this is okay to can be merged, but I don´t know what can do the previous commit, because is the same commit that you use in the PR "Maintenance Page: State, Mutations and Actions added ". We must talk about this.

**Comment by JuanJosePG** [comment]
Yes, its true. I didnt realise. Maybe I copied and pasted the commit after swap between branches. I dont know If it's a big problem.

File: src/App.vue [hunk 1]
```diff
@@ -18,6 +21,7 @@ import authStore from '@/store/auth-store/AuthStore';
 import WhatsappButton from '@/components/Contact/WhatsappButton.vue';
 import CallButton from '@/components/Contact/CallButton.vue';
 import Loading from '@/components/Utils/Loading.vue';
+import MaintenancePage from '@/views/MaintenancePage.vue';
```
Comment (Andresws12) [inline src/App.vue:24]: @JuanJosePG The view with the views and components with the components jaajaja

**Comment by Andresws12** [comment]
> Yes, its true. I didnt realise. Maybe I copied and pasted the commit after swap between branches. I dont know If it's a big problem.

@JuanJosePG Please change the titlte to understand better

File: src/App.vue (hunk 1 above)
Comment (JuanJosePG) [reply src/App.vue]: I didn't realise, sorry. The commit is done, check it whenever you can.

**Comment by JuanJosePG** [comment]
I forget to notify you that the commit and the title were done. @Andresws12

**Comment by Andresws12** [comment]
For me it's okay, @PVillasana @pedrojgm What do you think?

**Pull request merged**